### DISCUSSION TITLE
Timer to next Live event

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#248] [Feature requests] [answered]
Would be great to have a timer for when the next Live event is, as demonstrated in the picture below.

![image](https://user-images.githubusercontent.com/22590445/194561698-480e1305-181f-4a5a-a9f2-37c3a9938281.png)

**Comment by f1multiviewer** [comment, marked as answer]
Duplicate of https://github.com/f1multiviewer/issue-tracker/discussions/137
